tensorflow_serving/batching/batching_util.h

```diff
@@ -0,0 +1,206 @@
+/* Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Actually, if we pad to max length in batch then there are two methods: AddPadding and CalculateMaxDimSizes. I moved other padding functions to .cc and made changes to unit tests.